pages/linux/uuid.md

```diff
@@ -0,0 +1,33 @@
+# uuid
+
+> Generate and decode Universally Unique Identifiers (UUID).
+> See also `uuidgen`.
+> More information: <https://manned.org/uuid>.
+
+- Generate a v1 UUID (based on time and system's hardware address, if present):
+
+`uuid`
+
+- Generate a v4 UUID (random data based):
+
+`uuid -v {{4}}`
+
+- Generate a v5 UUID (name based) with an URL namespace:
+
+`uuid -v {{5}} ns:URL {{https://example.com}}`
```

Comment:
I have generalized the example with all my newly acquired knowledge:

```suggestion
- Generate a v5 UUID (name based) with a specified namespace:

`uuid -v {{5}} ns:{{DNS|URL|OID|X500}} {{https://example.com}}`
```

Reply:
Why does it has to be a URL? You can put any string you want in there. And why is there a URL namespace if you can use a URL with all other namespaces too?

If this is obvious or hard to explain with a few words, then the example is fine as is.

Reply:
@marchersimon yes, you can put any string there, there is no example of this here. But there are exactly for namepace *prefixes*, mentioned in the example. Does this read better?


```suggestion
- Generate a v5 UUID (name based) with a specified namespace prefix:

`uuid -v {{5}} ns:{{DNS|URL|OID|X500}} {{https://example.com}}`
```

Reply:
I think the wording is fine, I'm just confused about the command itself. If any string is possible, why don't we use `uuid -v {{5}} ns:{{DNS|URL|OID|X500}} {{string}}` or something like that?

Reply:
@marchersimon aah yes, you are right, nice catch! I have edited the suggestion here to match the one in the [uuidgen](https://github.com/tldr-pages/tldr/pull/5892/files#diff-e6eaf9f527253cf5204d3886c8e32d44bb74188debe51af664430522ba68cbfb) file